loops-and-arrays.js

```diff
@@ -23,30 +33,50 @@ const names = ["Henk", "Piet", "Fred", "Joop"];
 
 const numbers = [2, 4, 5, 29, 38];
 
+for (let i = 0; i < numbers.length; i++) {
+    if (numbers[i] % 2 === 0) {
+        numbers[i] = numbers[i] * 2;
+    } else {
+        numbers[i] = numbers[i] * 3;
+    }
+}
+
+console.log(numbers);
+
```

Comment:
Ik zie je aantekening staan en dacht volgens mij ook dat de verwachte uitkomst niet klopte.
Met onderstaande code krijg ik alleen toch wel die uitkomst. Ik zie even niet wat goed of fout is;

for (let i = 0; i < numbers.length; i++) {
    if (i % 2 === 0) {
        console.log(numbers[i] * 2);
    }
    else {
        console.log(numbers[i] * 3);
    }
}

Reply:
Wat gek! Want als je je mijn uitwerking in de terminal logt dan geeft het wel: [ 4, 8, 15, 87, 76 ]. Bij jou niet?

Reply:
Bij deze gevonden!!

for (let i = 0; i < numbers.length; i++) {
if (numbers[i] % 2 === 0) {
numbers[i] = numbers[i] * 2;
} else {
numbers[i] = numbers[i] * 3;
}
}
console.log(numbers); >>>> geeft [ 4, 8, 15, 87, 76 ]

&&

for (let i = 0; i < numbers.length; i++) {
if (i % 2 === 0) {
numbers[i] = numbers[i] * 2;
}
if (i % 2 !== 0) {
numbers[i] = numbers[i] * 3;
}

console.log(numbers); >>>> geeft [ 4, 12, 10, 87, 76 ]

Als je goed kijkt zie je het verschil! Is wel een puzzel hoor ;)